### DISCUSSION TITLE
Is there a way to edit default audit config policies?

### DISCUSSION THREAD

**Discussion by data-dude** [aquasecurity/trivy#5719] [Documentation]
### Description

I see the default rego policies are here:

https://github.com/aquasecurity/trivy-policies/tree/main/rules/kubernetes/policies/general

If I want to update them so that they exclude certain namespaces, how do i do that?

### Suggestions

Please provide detailed documentation on how to update the default trivy config audit policies.

**Comment by simar7** [comment]
hi @data-dude you could specify the `--namespace` flag.

`trivy k8s --namespace <yournamespace>`

**Reply by data-dude** [reply]
i installed trivy in trivy-system namespace using a helm chart.  And it's in there that a configmap is created but it's empty but i realized that is for custom policies.  What I want to do is modify the built in config audit policies.   Many of them create noise because we have many exceptions.  I'd like to add those exceptions to the built in policies.  At this point it seems the only way forward is to remove the built in policies and just add my own.  Though it'd be a lot nicer to just modify the existing ones.

**Reply by simar7** [reply]
cc @chen-keinan
